packages/server/graphql/public/rootSchema.ts

```diff
@@ -62,17 +70,19 @@ const {schema: withGitLabSchema, gitlabRequest} = nestGitLabEndpoint({
 const importAllStrings = (context: __WebpackModuleApi.RequireContext) => {
   return context.keys().map((id) => context(id).default)
 }
+
+// Types from SDL first
 const typeDefs = importAllStrings(require.context('./typeDefs', false, /.graphql$/))
 
-const withNestedSchema = mergeSchemas({
-  schemas: [withGitHubSchema, withGitLabSchema],
+const legacyParabolWithNestedSchema = mergeSchemas({
+  schemas: [legacyParabolWithGitHubSchema, legacyParabolWithGitLabSchema],
   typeDefs
 })
 
 // IMPORTANT! mergeSchemas has a bug where resolvers will be overwritten by the default resolvers
 // See https://github.com/ardatan/graphql-tools/issues/4367
-const withNestedResolversSchema = addResolversToSchema({
-  schema: withNestedSchema,
+const parabolWithNestedResolversSchema = addResolversToSchema({
+  schema: legacyParabolWithNestedSchema,
   resolvers: composeResolvers(resolvers, permissions)
```

Comment:
@mattkrick we're only applying permissions to SDL first resolvers. Is that intentional?

Reply:
It was, but I'm all for changing it if you prefer.
My logic was the old types use the old pattern. If we refactor the permissions into the new permissionMap then we should also refactor that resolver into the new SDL pattern, too.